charts/ceph-csi-cephfs/values.yaml

```diff
@@ -214,6 +214,11 @@ provisioner:
     ## https://github.com/kubernetes-csi/external-provisioner#command-line-options
     extraArgs: []
 
+    args:
+      # httpEndpoint enables the http server for diagnostics,
+      # health checks and metrics
```

Comment:
Maybe also mention that not providing the option disables it?

Reply:
IMO we should not have it enabled but disable by default (commenting the port) and let the user enable it when required